src/tribler-core/tribler_core/modules/metadata_store/community/tests/test_remote_query_community.py

```diff
@@ -252,7 +252,7 @@ async def test_process_rpc_query_match_one(self):
         results = await self.overlay(0).process_rpc_query(dumps({}))
         self.assertEqual(1, len(results))
 
-        channel_md, = results
+        (channel_md,) = results
```

Comment:
**question:** Just curious. What was the reason for this replacement?

Reply:
I guess to make it more explicit that `results` is a tuple?